### DISCUSSION TITLE
search file with finding file (`<space><space>`) but got "not .git directory"

### DISCUSSION THREAD

**Discussion by lainegates** [LazyVim/LazyVim#2418] [Q&A] [answered]
I try to find files under root directory with `<space><space>` .

But lazy.nvim tell me that "not a git directory"

![image](https://github.com/LazyVim/LazyVim/assets/5265648/4485621b-bf78-4060-be72-65d7b0214c0d)

Is there a way to disable the setting that the root directory must be a git directory?

**Comment by dpetka2001** [comment]
There is no such setting that the root directory has to be a git directory. Try the following
```bash
1. `mkdir ~/.config/lazytest`
2. `git clone https://github.com/LazyVim/starter ~/.config/lazytest`
3. `NVIM_APPNAME=lazytest nvim` and wait for the installation to complete
4. Restart Neovim with the command provided in step 3
```
Does the issue persist with the above commands?

Cleanup 
```bash
rm -rf ~/.config/lazytest
rm -rf ~/.local/share/lazytest
rm -rf ~/.local/state/lazytest
rm -rf ~/.cache/lazytest
```

**Reply by lainegates** [reply, marked as answer]
@dpetka2001   Thanks ! I checked carefully, and then find that 
![image](https://github.com/LazyVim/LazyVim/assets/5265648/5b46ea17-72ba-48b2-b77e-b18f179e81a3)

lazy.nvim will try to use `Telescope git_files` if find `.git` directory in root directory.
`Telescope git_files` failed because owner of the repo is `$USER in host`, not `root` in docker. While I start lazy.nvim in docker.
when I run `git config --global --add safe.directory $ROOT_DIRECTORY`, the lazy.nvim works great!
